dev-resources/keycloak_demo/test-realm.json

```diff
@@ -763,7 +763,7 @@
         "jsonType.label" : "String"
       }
     } ],
-    "defaultClientScopes" : [ "web-origins", "profile", "roles", "lrs:all", "email" ],
+    "defaultClientScopes" : [ "web-origins", "profile", "roles", "lrs:statements/write", "lrs:statements/read/mine", "email" ],
```

Comment:
I wonder what the implications would be for narrowing the default client scopes. (If it just affects the demo then I expect not many.)

Reply:
Yeah, it's just the demo, and this demonstrates OIDC better (I can only read/write my own statements)